docs/plugin-maintainers-guide.md

```diff
@@ -0,0 +1,54 @@
+# Plugin Maintainer Guide
+
+## Table of Contents
+
+- [Plugin Maintainer Guide](#plugin-maintainer-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Maintaining and patching an older release line](#maintaining-and-patching-an-older-release-line)
+    - [Patching an older release](#patching-an-older-release)
+
+## Maintaining and patching an older release line
+
+It may be necessary to patch a prior release line of a plugin when users depend on an older, but stable version and while a newer, incompatible, major version of the plugin exists. Typically for these older releases, only major bugs and security issues will need to be remediated. Not every plugin will need this workflow.
+
+This guide will describe the steps needed to release on an older version.
+
+### Patching an older release
+
+When patching an older release, follow the steps below to ensure the correct workflow is applied:
+
+1. Request a `workspace/${workspace}` branch by asking one of the @backstage/community-plugins-maintainers.
```

Comment:
Do you think there's any way to link to a list of the people in this group, so that users would be able to figure out who to ask? 

Alternatively a link to https://github.com/backstage/community/tree/main/sigs/sig-community-plugins#leadership could provide an example of some people you can ask

Reply:
Unfortunately as a non-member of the Backstage org this link isn't visible to me, so it'd be nicer to link to something the general public can access

Reply:
Really, I could swear I was able to do that in the past 🤔 

Well the other link is terribly out of date, this would probably be better then: https://github.com/backstage/backstage/blob/master/OWNERS.md#community-plugins